File: Python/Remove_Duplicates_from_Sorted_List.py
def remove_duplicates():
    arr = [1, 1, 2, 2, 3, 3, 4, 4, 5, 5, 6, 6, 6, 6, 6]
    arr1 = []

    for i in range(len(arr) - 1):
        if arr[i] == arr[i + 1]:
            if not arr1 or arr1[-1] != arr[i]:
                arr1.append(arr[i])

    print(arr1)

if __name__ == "__main__":
    remove_duplicates()

File: Remove_Duplicates_from_Sorted_List.java
void main() {
 int arr[] = {1, 1, 2, 2, 3, 3, 4, 4, 5, 5,6,6,6,6,6};

        List<Integer> arr1 = new ArrayList<>();

        for (int i = 0; i < arr.length - 1; i++) {

            if (arr[i] == arr[i + 1]) {

                if (arr1.isEmpty() || arr1.get(arr1.size() - 1) != arr[i]) {
                    arr1.add(arr[i]);
                }
            }
        }

        System.out.println(arr1);
    
}